framework/doc/content/source/postprocessors/TimeExtremeValue.md

```diff
@@ -2,7 +2,10 @@
 
 The TimeExtremeValue Postprocessor reports the extreme value of coupled Postprocessor seen over time.
 For example, this Postprocessor can be used to record the peak temperature or maximum stress in a
-simulation with an oscillating force.
+simulation with an oscillating force. The minimum, absolute minimum, and absolute maximum over time
+are also options in addition to the default maximum.
+
+There is a parameter `output_type` that allows the user to select whether to output the extreme value requested as described above or the time at which the extreme value occurred.
```

Comment:
Can you use the !param syntax  here so that this fails if the disappears?

Reply:
Sure.